laws/src/main/scala/cats/laws/discipline/TraverseFilterTests.scala

```diff
@@ -35,6 +35,7 @@ trait TraverseFilterTests[F[_]] extends FunctorFilterTests[F] {
     ArbFA: Arbitrary[F[A]],
     ArbFOA: Arbitrary[F[Option[A]]],
     ArbFABoo: Arbitrary[PartialFunction[A, B]],
+    ArbFAOB: Arbitrary[PartialFunction[A, Option[B]]],
```

Comment:
I don't think we can add this since we try to keep binary compatibility here. But I don't think we have to.
```scala
implicit val arbFAOB: Arbitrary[PartialFunction[A, Option[B]]]] =
  Abitrary(ArbFABoo.arbitrary.map { pfab =>
    { case a if pfab.isDefinedAt(a) =>
      val b = pfab(a)
      if (((a.hashCode ^ b.hashCode) & 1) == 1) Some(b)
      else None
    }
  }
```
and put it before new DefaultRuleSet below.

Reply:
Done